var/spack/repos/builtin/packages/tar/package.py

```diff
@@ -58,6 +68,8 @@ class Tar(AutotoolsPackage, GNUMirrorPackage):
     patch("nvhpc-1.34.patch", when="@1.34 %nvhpc")
     # Workaround bug where __LONG_WIDTH__ is not defined
     patch("nvhpc-long-width.patch", when="@1.34 %nvhpc")
```

Comment:
These two patches still need to be ported (or evaluated for need for porting).

Reply:
`nvhpc-long-width.patch` applies for 1.35, but `nvhpc-1.34.patch` only applies half (`gnu/intprops.h` context missing).

```suggestion
    patch("nvhpc-1.34.patch", when="@1.34 %nvhpc")
    # Workaround bug where __LONG_WIDTH__ is not defined
    patch("nvhpc-long-width.patch", when="@1.34: %nvhpc")
```

Reply:
@samcmill FYI I had to add a conflict for tar@1.35 here since the previous patches don't apply anymore and I have no ability to test with nvhpc.